requirements-dev-only-frozen.txt

```diff
@@ -0,0 +1,72 @@
+attrs==21.4.0
```

Comment:
It _might_ be useful to add a comment in this file that folks should use `make freeze-requirements` to update this file. We will need to update Sentry's dev env docs on how to update Python deps.

Ideally, we should have a check that fails if someone touches `requirements-dev.txt` but the frozen file is not modified.

Reply:
Yeah, I'll add one.